### ISSUE TITLE
Block requiements to move

### ISSUE THREAD

**Issue by Coder-TheSid** [APDevTeam/Movecraft#520]
**Is your feature request related to a problem? Please describe.**
On crafts a Block must be touch to Ship to move its like  in real life Train's need railway to move so a example a tram needs touchs to Iron block to move **But it wont fly too!**

**Comment by Coder-TheSid** [comment]
Note: this feature should only avaible for ground crafts

**Comment by Coder-TheSid** [comment]
Also i have a new feature idea: craftType, it will have 3 chooice Ground/Air/Water that could help us to make the ground/air and water crafts easyer.

**Comment by Galliut** [comment]
The first part is good, but I dislike the craftType idea as craft files have specific properties to allow for more creative versality. Craft files by themselves are "craft types".

**Comment by TylerS1066** [comment]
> The first part is good, but I dislike the craftType idea as craft files have specific properties to allow for more creative versality. Craft files by themselves are "craft types".

I agree, I think it best that you just copy/paste from the defaults.
